app/views/admin_users/sessions/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= page_template page_title: t('.heading'), back_link: :none do %>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="govuk-form-group">
+      <%= f.label :username, class: 'govuk-label' %>
+      <%= f.text_field :username, for: 'width-15', autofocus: true, class: 'govuk-input govuk-input--width-20' %>
```

Comment:
what does the `for: 'width-15'` do?

Reply:
Ah - good spot - some code I put in by mistake and forgot to remove.